catris.py

```diff
@@ -530,11 +530,6 @@ def _start_playing(self) -> None:
                 for client in self._client.server.clients
                 if client.name is not None
             }
-            names_in_use = names_of_connected_players | {p.name for p in state.players}
```

Comment:
This has problems, if a user leaves and then a different named user is allowed to join